pkg/mysql/client.go

```diff
@@ -1164,7 +1244,7 @@ func (conn *BackendConnection) ReadColumnDefinitions() ([]proto.Field, error) {
 }
 
 // ExecuteWithWarningCountIterRow is for fetching results and a warning count
-// Note: In a future iteration this should be abolished and merged into the Execute API.
+// Note: In a future iteration this should be abolished and merged into to Execute API.
```

Comment:
// Note: In a future iteration this should be abolished and merged into the Executed API.

maybe this ?